internal/actions/update.go

```diff
@@ -36,12 +38,14 @@ func Update(client container.Client, params UpdateParams) error {
 	checkDependencies(containers)
 
 	if params.MonitorOnly {
+		executePostCheck(client, params)
```

Comment:
This is a bit weird, isn't it? monitor only is supposed to not run anything that might mutate the containers.
Both `executePreCheck` and `executePostCheck` should be skipped if the user runs watchtower with `--monitor-only` enabled in my opinion.

Reply:
I get your point. One could also see them as independent of each other as you need to activate hooks explicitly. Hooks also don't necessarily have to mutate any container and it could be unnecessarily limiting just to block them for that reason. If we change so that the hooks is skipped on "--monitor-only" I suggest that a warning of some kind is thrown when combined with "--enable-lifecycle-hooks".